beacon-chain/blockchain/log.go

```diff
@@ -80,3 +81,23 @@ func logBlockSyncStatus(block interfaces.BeaconBlock, blockRoot [32]byte, justif
 	}).Info("Synced new block")
 	return nil
 }
+
+// logs payload related data every slot.
+func logPayload(block interfaces.BeaconBlock) error {
+	payload, err := block.Body().ExecutionPayload()
```

Comment:
We only need to do this for bellatrix block

```suggestion
func logPayload(block interfaces.BeaconBlock) error {
        isExecutionBlk, err := blocks.IsExecutionBlock(block.Body())
	if err != nil {
		return errors.Wrap(err, "could not determine if block is execution block")
	}
	if !isExecutionBlk {
		return nil, nil
	}
	payload, err := block.Body().ExecutionPayload()
```

Reply:
Only Bellatrix block is execution block. My previous statement meant we need to do this to account for non bellatrix block because they don't have payload fields

Reply:
Understood, thank you!